xml/System.Security.Cryptography.Pkcs/SubjectIdentifier.xml

```diff
@@ -52,10 +52,21 @@
         <Parameter Name="certificate" Type="System.Security.Cryptography.X509Certificates.X509Certificate2" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="certificate">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="certificate">The certificate to verify for a match.</param>
+        <summary>Verifies if the specified certificate was used for the decryption operation.</summary>
+        <returns><see langword="true" /> if the certificate; <see langword="false" /> otherwise.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the specified certificate was used for decryption; otherwise, <see langword="false" />.</returns>
```
The return value information is incomplete.